app/assets/javascripts/discourse/app/services/search.js

```diff
@@ -21,6 +21,14 @@ export default class Search extends Service {
     return this.searchContext?.type || null;
   }
 
+  updateActiveGlobalTerm(term) {
+    this.activeGlobalSearchTerm = term;
+
+    // This appEvent allows themes/plugins to listen for search term changes without relying on
+    // an observer.
+    this.appEvents.trigger("search:term-updated", { term });
```

Comment:
Can't you import the service and do:

```javascript
get something() {
  return "current search: " + this.search.activeGlobalSearchTerm;
}
```

Reply:
If you need to do something on the DOM with this you can do

```
myModifier = modifier(element => {
  element.focus();
})

<tempalte>
  <div {{this.myModifier this.search.activeGlobalSearchTerm}}></div>
</template>
```

I think that should work, syntax might be wrong, but something similar will work.